packages/common-components/src/Breadcrumb/Breadcrumb.tsx

```diff
@@ -96,23 +114,29 @@ const Breadcrumb: React.FC<BreadcrumbProps> = ({
   crumbs = [],
   homeOnClick,
   hideHome,
+  homeRef,
+  homeActive,
   className,
   showDropDown
 }: BreadcrumbProps) => {
   const classes = useStyles()
 
   const generateFullCrumbs = (crumbs: Crumb[]) => {
-    return crumbs.map((item: Crumb, index: number) => (
+    return crumbs.map((crumb: Crumb, index: number) => (
       <Fragment key={`crumb-${index}`}>
-        <div>
+        {!crumb.component ?  <div
+          ref={crumb.forwardedRef}
+          className={clsx(crumb.active && "active", classes.wrapper)}
+        >
           <Typography
-            onClick={() => (item.onClick ? item.onClick() : null)}
-            className={clsx(classes.crumb, item.onClick && "clickable")}
+            onClick={() => (crumb.onClick ? crumb.onClick() : null)}
```

Comment:
No need for the ternary here either
```suggestion
            onClick={() => crumb.onClick ? crumb.onClick()}
```